### PR TITLE
Migrate - Swift 5.7 and update deployment target to iOS16

### PR DESCRIPTION
**Pull request by Kedar-27**
Hi, I have update the code files and done necessary changes for the project to run on iOS 16

### CODE REVIEW

**Comment by gokulrajanpillai** [comment]
@Kedar-27 Thanks for the update!
I would recommend not checking in the Pods folder. Was anything in the Pod module QRCodeReader modified after installing it?
Else we can update .gitignore to ignore the Pods folder.

**Comment by Kedar-27** [comment]
Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7

**Comment by gokulrajanpillai** [comment]
> Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7

Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder.
Just wanted to know why was the Pods folder included in the PR?

**Comment by Kedar-27** [comment]
> > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> 
> Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?

It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore

**Comment by Kedar-27** [comment]
> > > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> > 
> > 
> > Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?
> 
> It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore

Done.. Please check

File: .gitignore [hunk 1]
```diff
@@ -40,6 +41,9 @@ xcuserdata/
 ## Carthage
 Carthage/Build
 
+# Cocoapods
+Pods/
```
Comment (gokulrajanpillai) [inline .gitignore:45]: Awesome 🔥

**Comment by gokulrajanpillai** [comment]
> > > > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> > > 
> > > 
> > > Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?
> > 
> > 
> > It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore
> 
> Done.. Please check

Thanks a lot. Can you also make a commit removing the Pods folder? it is still part of the pull request

**Review by gokulrajanpillai — CHANGES_REQUESTED** [review]
Once the Pods folder is removed, we can squash merge to remove it from the history (y)

**Comment by Kedar-27** [comment]
> > > > > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> > > > 
> > > > 
> > > > Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?
> > > 
> > > 
> > > It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore
> > 
> > 
> > Done.. Please check
> 
> Thanks a lot. Can you also make a commit removing the Pods folder? it is still part of the pull request

Yes I have removed it now.

**Review by gokulrajanpillai — APPROVED** [review]

**Comment by Kedar-27** [comment]
Thanks for accepting.

**Comment by gokulrajanpillai** [comment]
Thank you for your contribution! 😃

**Pull request merged**